lib/order_form.rb

```diff
@@ -0,0 +1,37 @@
+require_relative 'takeaway'
+require_relative 'menu'
+
+class OrderForm
+
+  attr_reader :order 
+  attr_accessor :takeaway
+
+  def initialize
+    @order = []
+    @dishes = Menu.new.dishes
+    @takeaway = Takeaway.new
+  end
+
+  def show_dishes
+    @dishes.map{ |k,v| "#{k} (£#{'%.2f' % v})"}.join(', ')
```

Comment:
Space missing after comma.<br>Space missing to the left of {.<br>Space missing inside }.